### ISSUE TITLE
Reverse indenting algorithm could be done on every leading space rather than groups of 4 spaces

### ISSUE THREAD

**Issue by james5889010** [robert-brennan/reverseindent#4]
If the algorithm were to be based on each leading space at the start of a line rather than groups of 4 leading spaces the application would still work equally well for developers who use text editors with different tab group lengths.

**Comment by robert-brennan** [comment]
Good idea. Want to try it?

**Comment by james5889010** [comment]
Yes, I will give it a go on my fork (https://github.com/james5889010/reverseindent). 

I'm also working on copy and paste functionality. Is there anywhere I can deploy the app for design review? I have it available for a quick view here on jsfiddle: https://jsfiddle.net/f25320Lq/. Just click Run and make the Results box as large as possible.

**Comment by james5889010** [comment]
Pull request for issue title here: https://github.com/robert-brennan/reverseindent/pull/5
